mu4e/mu4e-view-gnus.el

```diff
@@ -37,6 +37,27 @@
 (defvar helm-comp-read-use-marked)
 (defvar mu4e~view-rendering nil)
 
+(defvar mu4e-view-blocked-images "."
+  "Regexp matching image URLs to be blocked, or nil meaning not
```

Comment:
Can you make it a defcustom, mirroring the gnus one?

Reply:
Done.